components/OutcropDetailsPage/index.js

```diff
@@ -94,8 +95,9 @@ const OutcropDetailsPage = () => {
   };
 
   const setStoredFieldTrips = (data) => {
+    //function to set stored field trips in local storage
     try {
-      localStorage.setItem("fieldTrips", JSON.stringify(data)); // set stored field trips
+      localStorage.setItem("fieldTrips", JSON.stringify(data));
```

Comment:
for your local storage usage - I would suggest to use [use-local-storage-state](https://github.com/neuefische/muc-web-23-3/blob/main/sessions/react-with-local-storage/react-with-local-storage.md#use-local-storage-state) hook. check it out think about it (could be a later userstory)